cryptofeed/exchange/ftx.py

```diff
@@ -44,8 +45,28 @@ def _parse_symbol_data(cls, data: dict, symbol_separator: str) -> Tuple[Dict, Di
             symbol = d['name']
             ret[normalized] = symbol
             info['tick_size'][normalized] = d['priceIncrement']
+            
+            # Add index symbol for non-spot markets
+            if d['underlying'] is None or d['type'] == 'spot':
+                continue
+            index_normalized = cls._translate_index_symbol(d['underlying'], False)
+            ret[index_normalized] = index_normalized
+            info['index_to_derivative'][index_normalized] = symbol
+            info['is_index'][index_normalized] = True
         return ret, info
 
+    @classmethod
+    def _index_to_derivative(cls, symbol: str):
+        """
+        Returns a derivative (e.g. future, perpetual) associated with the given index symbol,
+        or None if the given symbol is not an index symbol.
+        """
+        return cls.info()['index_to_derivative'].get(symbol, None)
```

Comment:
can we skip using classmethods here to avoid extra complexity and noise when merging upstream

`cls.info()['index_to_derivative'].get(symbol)`

Reply:
just doesn't need to be a function at all

in the 1 place you use it, just do `derivative = cls.info()['index_to_derivative'].get(symbol)`